p2p/datasets.go

```diff
@@ -45,14 +48,22 @@ func (n *QriNode) RequestDatasetsList(ctx context.Context, pid peer.ID, p Datase
 
 	req = req.WithHeaders("phase", "request")
 
-	replies := make(chan Message)
-	err = n.SendMessage(ctx, req, replies, pid)
+	s, err := n.host.NewStream(ctx, pid, depQriProtocolID)
 	if err != nil {
-		log.Debug(err.Error())
-		return nil, fmt.Errorf("send dataset info message error: %s", err.Error())
+		return nil, fmt.Errorf("error opening stream: %s", err.Error())
```

Comment:
Note: replacing mentions of `n.SendMessage`. It relies on code that is already removed & is way overkill for a situation like this, where all you are doing is directly asking one peer for a response.